tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
```

Comment:
Just so I understand (from the other CR).

project.json is for iOS apps that run through vsimporter.

packageable.project.json is for iOS middleware that runs through vsimporter.  So, we need to include winobjc.packaging for iOS middleware to create a nuget package.